oak_functions/lookup/src/lib.rs

```diff
@@ -219,20 +285,24 @@ mod tests {
         let lookup_data_0 = manager.create_lookup_data();
         assert_eq!(lookup_data_0.len(), 0);
 
-        manager.update_data(HashMap::from_iter(
-            [(b"key1".to_vec(), b"value1".to_vec())].into_iter(),
-        ));
+        manager.update_data(
+            UpdateAction::StartAndFinish,
+            HashMap::from_iter([(b"key1".to_vec(), b"value1".to_vec())].into_iter()),
+        );
         let lookup_data_1 = manager.create_lookup_data();
         assert_eq!(lookup_data_0.len(), 0);
         assert_eq!(lookup_data_1.len(), 1);
 
-        manager.update_data(HashMap::from_iter(
-            [
-                (b"key1".to_vec(), b"value1".to_vec()),
-                (b"key2".to_vec(), b"value2".to_vec()),
-            ]
-            .into_iter(),
-        ));
+        manager.update_data(
+            UpdateAction::StartAndFinish,
+            HashMap::from_iter(
+                [
+                    (b"key1".to_vec(), b"value1".to_vec()),
```

Comment:
You could use maplit here for readability, but up to you.

Reply:
I am not sure, because this is a no_std Hashmap.